### PR TITLE
Feature/netstandard migration

### PR DESCRIPTION
**Pull request by linkdotnet**

### CODE REVIEW

File: source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj [hunk 1]
```diff
@@ -1,133 +1,41 @@
...
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{CB3FE911-7034-471C-A2D7-CB89842423FC}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Jobbr.WebApi.Tests</RootNamespace>
-    <AssemblyName>Jobbr.WebApi.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net462</TargetFramework>
```
Comment (eXpl0it3r) [inline source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj:4]: Any reason not to target netstandard as well?

Comment (linkdotnet) [reply source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj:4]: Damn it :D toooooo many projects ;)

**Comment by chumachun** [comment]
LGTM, well done. Thanks. Much appreciated.

File: source/Sanbox.JobRunner/Sandbox.JobRunner.csproj [hunk 2]
```diff
@@ -1,60 +1,24 @@
...
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{C75CE4B3-EF65-45B8-A1D2-E5D5CA6F1157}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>Sandbox.JobRunner</RootNamespace>
-    <AssemblyName>Sandbox.JobRunner</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net472</TargetFramework>
```
Comment (eXpl0it3r) [inline source/Sanbox.JobRunner/Sandbox.JobRunner.csproj:5]: In that case, check here as well 😉

File: source/Sandbox.JobServer/Sandbox.JobServer.csproj [hunk 3]
```diff
@@ -1,107 +1,41 @@
...
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{28B3BFFA-26E2-4A0E-A329-B6FD729FB981}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>Sandbox.JobServer</RootNamespace>
-    <AssemblyName>Sandbox.JobServer</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net472</TargetFramework>
```
Comment (eXpl0it3r) [inline source/Sandbox.JobServer/Sandbox.JobServer.csproj:5]: What about this? 🤔

File: source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj (hunk 1 above)
Comment (linkdotnet) [reply source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj:4]: Test projects have to be netcore ;) but will chagne that

File: source/Sandbox.JobServer/Sandbox.JobServer.csproj (hunk 3 above)
Comment (linkdotnet) [reply source/Sandbox.JobServer/Sandbox.JobServer.csproj:5]: These are just "executable" samples which don't necessary have to be upgrade.
But I can upgrade them to netcoreapp2.0

File: source/Sanbox.JobRunner/Sandbox.JobRunner.csproj (hunk 2 above)
Comment (linkdotnet) [reply source/Sanbox.JobRunner/Sandbox.JobRunner.csproj:5]: These are just "executable" samples which don't necessary have to be upgrade.
But I can upgrade them to netcoreapp2.0

**Comment by linkdotnet** [comment]
I will let the sandbox application be on net462 as migration to .netcoreapp3.1 throws some errors locally

**Comment by linkdotnet** [comment]
@eXpl0it3r Test Project is still depending on Jobbr.Server which is .NETFramework 4.5
Therefore the test project will stay .net framework instead of .netcore3.1

**Review by eXpl0it3r — APPROVED** [review]

**Pull request merged**